Apps/WebClient/src/WebClient.csproj

```diff
@@ -15,7 +15,14 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DocumentationFile>.\WebClient.xml</DocumentationFile>
-  </PropertyGroup>  
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Listeners\**" />
+    <Content Remove="Listeners\**" />
+    <EmbeddedResource Remove="Listeners\**" />
+    <None Remove="Listeners\**" />
```

Comment:
Not sure what all of these are about, seems like we want to remove everything from Listeners , but in case VS is removing them and you are not aware of.

Reply:
The folder didn't exist but was still in the project file.  This is just cleanup.